groupAnagrams.js

```diff
@@ -0,0 +1,44 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/group-anagrams/
+
+const groupAnagrams = (strs) => {
+  let strsArray = strs;
+  const result = [];
+  while (strsArray.length > 0) {
+    const grouppedAnagrams = [strsArray[0]]; // "eat"
+    for (let j = 1; j < strsArray.length; j += 1) {
+      if (strsArray[0].length !== strsArray[j].length) {
+        continue;
+      }
+      let letters = strsArray[j].split(""); // t e a
+      let strsArrItem = strsArray[0].split(""); // e a t
```

Comment:
`strsArrItem` doesn't tell me much about what it represents - consider renaming to something more meaningful.